custom/theme.scss

```diff
@@ -1,6 +1,6 @@
-$light-background: #e9e5fb;
```

Comment:
Consider using more descriptive variable names for background colors to improve code clarity.